tests/benchmarks/benchmark.py

```diff
@@ -1,10 +1,52 @@
+import numpy as np
 import os
 import pytest
 
 from desc.__main__ import main
 from desc.grid import ConcentricGrid
 from desc.basis import FourierZernikeBasis
 from desc.transform import Transform
+from desc.equilibrium import EquilibriaFamily
+from desc.objectives import get_equilibrium_objective, get_fixed_boundary_constraints
+from desc.perturbations import perturb
+
+
+@pytest.fixture(scope="session")
+def TmpDir(tmpdir_factory):
+    """Create a temporary directory to store testing files."""
+    dir_path = tmpdir_factory.mktemp("test_results")
+    return dir_path
+
+
+@pytest.fixture(scope="session")
+def SOLOVEV(tmpdir_factory):
+    """Run SOLOVEV example."""
+    input_path = "..//tests//inputs//SOLOVEV"
+    output_dir = tmpdir_factory.mktemp("result")
+    desc_h5_path = output_dir.join("SOLOVEV_out.h5")
+    desc_nc_path = output_dir.join("SOLOVEV_out.nc")
+    vmec_nc_path = "..//tests//inputs//wout_SOLOVEV.nc"
+    booz_nc_path = output_dir.join("SOLOVEV_bx.nc")
+
+    cwd = os.path.dirname(__file__)
+    exec_dir = os.path.join(cwd, "..")
+    input_filename = os.path.join(exec_dir, input_path)
+
+    print("Running SOLOVEV test.")
+    print("exec_dir=", exec_dir)
+    print("cwd=", cwd)
+
+    args = ["-o", str(desc_h5_path), input_filename, "--numpy", "-vv"]
+    main(args)
+
+    SOLOVEV_out = {
+        "input_path": input_path,
+        "desc_h5_path": desc_h5_path,
+        "desc_nc_path": desc_nc_path,
+        "vmec_nc_path": vmec_nc_path,
+        "booz_nc_path": booz_nc_path,
+    }
+    return SOLOVEV_out
```

Comment:
This looks like it is duplicating code from `DESC/tests/conftest.py`. Can we reference that existing code, or does it need to also be included in the benchmark directory for some reason?

Reply:
Because the benchmarks are in their own folder, pytest does't know about ``conftest.py`` in the parent directory, so we need to include this here as well.